cmd/client.go

```diff
@@ -331,6 +331,17 @@ func (config *Config) isTLS() bool {
 	return isHostTLS(config)
 }
 
+type headerRoundTripper struct {
+	http.RoundTripper
```

Comment:
Add `customHeaders map[string][]string` to this structure instead of using globals.